src/workerd/server/workerd-api.c++

```diff
@@ -344,7 +344,6 @@ kj::Array<kj::String> getPythonRequirements(const Worker::Script::ModulesSource&
 struct WorkerdApi::Impl final {
   kj::Own<CompatibilityFlags::Reader> features;
   capnp::List<config::Extension>::Reader extensions;
-  kj::Maybe<kj::Own<jsg::modules::ModuleRegistry>> maybeOwnedModuleRegistry;
```

Comment:
Reviewer note: all of the changes in this file relate to changing the ownership of the `ModuleRegistry` instance from `Worker::Api` to `Worker::Script`